src/main/kotlin/fi/metatavu/timebank/api/keycloak/KeycloakController.kt

```diff
@@ -48,6 +49,44 @@ class KeycloakController {
         }
     }
 
+    /**
+     * Gets unspentVacations attribute for Person
+     * If not set will set and return default value of -1
+     *
+     * @param user UserRepresentation
+     * @return Int unspentVacations
+     */
+    fun getUsersUnspentVacationDays(user: UserRepresentation): Int {
+        return try {
+            user.attributes["unspentVacations"]!!.first()!!.toInt()
+        } catch (e: Exception){
+            updateUsersUnspentVacationDays(user, -1)
```

Comment:
Is this actually a good idea to set it to -1. Why not call the calculation function here to return a "default" calculated value?

Reply:
`VacationUtils::getPersonsVacations` should return the "almost correct" values.